crates/pallet-domains/src/tests.rs

```diff
@@ -1133,3 +1181,287 @@ fn test_type_with_default_nonce_encode() {
+                bundle_version: MockBundleVersion::V1,
+                execution_receipt_version: MockExecutionReceiptVersion::V1,
+            },
+            // this version at this block must exist due to the change in version
+            // in the next upgrade
+            true,
+        ),
+        // version change
+        // version from 156..160
+        (
+            160u32,
+            MockBundleAndExecutionReceiptVersion {
+                bundle_version: MockBundleVersion::V2,
+                execution_receipt_version: MockExecutionReceiptVersion::V1,
+            },
+            // this version at this block must exist due to the change in version
+            // in the next upgrade
+            true,
+        ),
+        // version change
+        // version from 161..200
+        (
+            200u32,
+            MockBundleAndExecutionReceiptVersion {
+                bundle_version: MockBundleVersion::V2,
+                execution_receipt_version: MockExecutionReceiptVersion::V2,
+            },
+            // this version at this block will not exist since next upgrade
+            // carries same version
+            false,
+        ),
+        // same version
+        // version from 201..250
+        (
+            250u32,
+            MockBundleAndExecutionReceiptVersion {
+                bundle_version: MockBundleVersion::V2,
+                execution_receipt_version: MockExecutionReceiptVersion::V2,
+            },
+            // this version at this block will not exist since next upgrade
+            // carries same version
+            false,
+        ),
+        // same version
+        // version from 251..300
+        (
+            300u32,
+            MockBundleAndExecutionReceiptVersion {
+                bundle_version: MockBundleVersion::V2,
+                execution_receipt_version: MockExecutionReceiptVersion::V2,
+            },
+            // this version at this block must exist due to the change in version
+            // in the next upgrade
+            true,
+        ),
+    ]
+}
+
+fn get_mock_version_queries() -> Vec<(u32, MockBundleAndExecutionReceiptVersion)> {
+    vec![
+        // version from 0..100
+        (
+            90u32,
+            MockBundleAndExecutionReceiptVersion {
+                bundle_version: MockBundleVersion::V0,
+                execution_receipt_version: MockExecutionReceiptVersion::V0,
+            },
+        ),
+        (
+            100u32,
+            MockBundleAndExecutionReceiptVersion {
+                bundle_version: MockBundleVersion::V0,
+                execution_receipt_version: MockExecutionReceiptVersion::V0,
+            },
+        ),
+        // version change
+        // version from 101..130
+        (
+            101u32,
+            MockBundleAndExecutionReceiptVersion {
+                bundle_version: MockBundleVersion::V1,
+                execution_receipt_version: MockExecutionReceiptVersion::V0,
+            },
+        ),
```

Comment:
`bundle_version` is changed to `MockBundleVersion::V1` at block `#121`, why do blocks before `#121` using `MockBundleVersion::V1`?

Reply:
```
MockBundleAndExecutionReceiptVersion {
                bundle_version: MockBundleVersion::V1,
                execution_receipt_version: MockExecutionReceiptVersion::V0,
            },
```

Bundle version changed from V0 to V1 from #101 until its changed again in #156

So bundle version is v0 until 0..100 and v1 from 101..155

Er version is changed from V0 to V1 from #131 
Do Er version is V0 until #130 and V1 until #161